lib/Doctrine/DBAL/Types/Type.php

```diff
@@ -305,6 +305,7 @@ public function canRequireSQLConversion()
      * @param \Doctrine\DBAL\Platforms\AbstractPlatform $platform
      *
      * @return string
+     * @deprecated implement WellKnownTextConverter::convertFromWellKnownTextSQL instead
```

Comment:
Should be removed in doctrine 3

Reply:
Not implementing the new interface though, which is optional and is not needed for any of the native types.

Reply:
Right, but existing types that do this type of conversion should implement it, if there are any (maybe not).